code-of-conduct.md

```diff
@@ -1,23 +1,27 @@
-# Code of Conduct
+# Woodcrest Code of Conduct
 
-1. **Embrace Diversity**: Woodcrest is a melting pot of roleplay experiences, languages, and styles. Whether you're a seasoned RPer or new to the scene, lead by example, offering guidance without elitism.
+Welcome to Woodcrest, a community where diversity, creativity, and respect form the bedrock of our shared experience. As members of this vibrant world, it is crucial that we adhere to a code of conduct and sim rules that uphold the integrity and enjoyment of all participants. Here is our comprehensive guide to ensuring a positive, inclusive, and enriching environment for everyone:
 
-1. **Foster Openness**: Engaging in local chat helps include everyone. While private messages (IMs) have their place, especially for welcoming those less inclined to speak up, always respect privacy and consent to avoid harassment.
+1. **Embrace and Respect Diversity**: Woodcrest is a diverse mosaic of roleplay experiences and backgrounds. We champion inclusivity and encourage all members to lead by example, fostering an environment where guidance is given with respect, not elitism.
 
-1. **Honor Boundaries**: Consent is key in roleplay. Avoid forcing unwelcome scenarios on others. Public events require participation by presence, but remember to seek approval for any significant environmental impact. Keep sensitive roleplay private and use discretion in public spaces to ensure comfort for all.
+1. **Communication is Key**: Open and respectful communication, primarily through local chat, is encouraged to maintain inclusivity. While private messages have their place, consent and privacy must always be respected to prevent harassment.
 
-1. **Minimize Drama**: Approach conflicts constructively and respect the community by avoiding slander and toxicity. Strive for peaceful resolutions and maintain a positive environment.
+1. **Consent and Boundaries**: Consent is the cornerstone of interaction here. Avoid imposing scenarios on others without their agreement. Public roleplay should not infringe on community comfort, and sensitive content must be approached with discretion. We strictly prohibit excessive violence and discriminatory language, both in public and private settings.
 
-1. **Enhance, Don’t Replace**: Use shouts, dances, and gestures to complement, not overshadow, roleplay. Excessive or disruptive gestures, along with out-of-character shouts, are not permitted. Griefing, trolling, and unsolicited promotions are grounds for disciplinary action.
+1. **Drama and Disruptions**: Constructive conflict resolution is essential. Slander, toxicity, and behaviors that disturb the peace are not tolerated. Scheduled events should not be interrupted without prior consent, emphasizing respect for the planning and effort involved.
 
-1. **Respect Events**: Do not interrupt scheduled events without prior agreement with the organizers. Understand the effort behind event planning and maintain a collaborative spirit.
```

Comment:
We changed this somewhat during a RPWG meeting where it falls on using common sense in a realistic setting. Which is also confusing.

I had some thoughts on this:

**Respect Events:** Events are a cornerstone of our community, and they require significant time, effort, and coordination to plan and execute. To ensure that events run smoothly and that everyone has an enjoyable experience, we ask that you adhere to the following guidelines:

1. Do not intentionally disrupt or derail scheduled events without prior agreement from the event organizers. This includes actions that would require a significant shift in focus away from the event's intended purpose, such as staging a medical emergency or initiating a major conflict.

2. However, we recognize that unexpected and spontaneous roleplay can add to the realism and immersion of our community. If a situation arises during an event that would realistically call for a change in focus, such as a minor injury or a brief interruption, feel free to proceed with the roleplay as long as it does not significantly derail the event.

3. If you plan to introduce a more substantial or prolonged interruption to an event, please first discuss your idea with the event organizers out of character (OOC). They can work with you to find a way to incorporate your roleplay in a manner that doesn't disrupt the event's flow or undermine the efforts of the organizers and participants.

4. Avoid using OOC influence to manipulate events or other characters' actions. Roleplay should be driven by in-character motivations and decisions.

5. If an unexpected interruption occurs during an event, the event organizers have the discretion to decide how to proceed. They may choose to incorporate the interruption into the event, or they may ask participants to pause the interruption until the event has concluded.

Remember, the goal is to balance spontaneity and realism with respect for the time and effort invested in event planning. By communicating openly and working together, we can create a collaborative and engaging roleplay experience for everyone involved. If you have any questions or concerns about event interruptions or this policy, please reach out to the moderation team for guidance.

Reply:
Accounted for in 083ca5ae6200179ab46fc45830f967d3d37ce86e